### PR TITLE
[Docs] (CodeCamp #76) translate add_transforms.md and conventions.md

### PR DESCRIPTION
**Pull request by Caczhtus**
Thanks for your contribution and we appreciate it a lot. The following instructions would make your pull request more healthy and more easily get feedback. If you do not understand some items, don't worry, just make the pull request and seek help from maintainers.

## Motivation

I hope to use this PR to make the project's documents more readable.

## Modification

I translated two documents, add_transforms.md and conventions.md.

## BC-breaking (Optional)

Does the modification introduce changes that break the backward compatibility of the downstream repositories?
If so, please describe how it breaks the compatibility and how the downstream projects should modify their code to keep compatibility with this PR.

No, no changes other than translations have been made.

## Use cases (Optional)

If this PR introduces a new feature, it is better to list some use cases here and update the documentation.

I translated two documents, add_transforms.md and conventions.md.

## Checklist

**Before PR**:

- [ ] Pre-commit or other linting tools are used to fix the potential lint issues.
- [ ] Bug fixes are fully covered by unit tests, the case that causes the bug should be added in the unit tests.
- [ ] The modification is covered by complete unit tests. If not, please add more unit test to ensure the correctness.
- [ ] The documentation has been modified accordingly, like docstring or example tutorials.

**After PR**:

- [ ] If the modification has potential influence on downstream or other related projects, this PR should be tested with those projects, like MMDet or MMSeg.
- [ ] CLA has been signed and all committers have signed the CLA in this PR.

### CODE REVIEW

File: docs/zh_cn/advanced_guides/add_transforms.md [hunk 1]
```diff
@@ -1 +1,118 @@
-# Add Transforms
+# 添加转换
```
Comment (fangyixiao18) [inline docs/zh_cn/advanced_guides/add_transforms.md]: ‘转换’ 翻译成 ‘数据变换’ 吧，意思上更好理解一点，全文 check 一下

File: docs/zh_cn/advanced_guides/add_transforms.md [hunk 2]
```diff
@@ -1 +1,118 @@
...
+
+在本教程中, 我们将介绍创建自定义转换的基本步骤。在学习创建自定义转换之前, 建议先了解文件 [transforms.md](transforms.md) 中转换的基本概念。
+
+- [添加转换](#add-transforms)
```
Comment (fangyixiao18) [inline docs/zh_cn/advanced_guides/add_transforms.md]: 目录这边小括号里的内容要和标题文字一致，空格 用  -  代替，不然跳转会有问题

**Review by fangyixiao18 — CHANGES_REQUESTED** [review]

File: docs/zh_cn/advanced_guides/add_transforms.md (hunk 1 above)
Comment (Caczhtus) [reply docs/zh_cn/advanced_guides/add_transforms.md]: 好的，已修改👌

File: docs/zh_cn/advanced_guides/add_transforms.md (hunk 2 above)
Comment (Caczhtus) [reply docs/zh_cn/advanced_guides/add_transforms.md]: 已修改👌

**Review by fangyixiao18 — APPROVED** [review]

**Pull request merged**